### ISSUE TITLE
Fix Marketo timeout issues

### ISSUE THREAD

**Issue by graceyin1218** [marklogic/eauser-geomapper#71]
First of all, Marketo is _really_ slow at returning relevant data... they should use MarkLogic 😉 

Secondly, the fact that it is taking so long (over an hour) is causing lots of errors when we go back to 2/2/2016 (right before EA1 was released) to sift through all the documents and pick out the EA users. Marketo successfully returns anywhere from 1 to 5 batches, and then says that my request has expired... Even when I update the timestamp every time I request a new batch...

(Marketo works fine for daily updates though. That usually takes only half an hour.)

TL;DR - Marketo is timing out, and I don't know how to work around this... Will have to ask various people again...

**Comment by graceyin1218** [comment]
Exhibit A: 

```xml
<?xml  version="1.0" encoding="UTF-8"?>
<SOAP-ENV:Envelope xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/">
<SOAP-ENV:Body>
<SOAP-ENV:Fault>
<faultcode>SOAP-ENV:Client</faultcode>
<faultstring>20016 - Request expired</faultstring>
<detail>
<ns1:serviceException xmlns:ns1="http://www.marketo.com/mktows/">
<name>mktServiceException</name>
<message>Request timestamp error (20016)</message>
<code>20016</code>
</ns1:serviceException>
</detail>
</SOAP-ENV:Fault>
</SOAP-ENV:Body>
</SOAP-ENV:Envelope>
```

**Comment by graceyin1218** [comment]
Turns out fn.currentDateTime() tells you when the query started... not the actual current time. Welp.
